app/Http/Controllers/User/PostScheduling/PostSchedulingController.php

```diff
@@ -34,8 +36,11 @@ function index(Request $request)
 
     public function uploadPostMedia(UploadPostMediaRequest $request)
     {
-        // dd($request->all(), explode(',', $request->tags));
-        $tags = implode(",", preg_filter('/^/', '#', explode(',', $request->tags)));
+        if ($request->tags != "") {
+            $tags = implode(" ", preg_filter('/^/', '#', explode(',', $request->tags)));
+        }else {
+            $tags="";
```

Comment:
Remove else and move this variable to before if

Reply:
Changes done